sandbox.html

```diff
@@ -572,6 +580,25 @@
   </div>
 </template>
 
+<template data-action="fetchObjectInfo" data-common="layer,object-ids,filter">
+  <div>
+    <label for="embeddingTargetId">Embedding target ID *</label>
+    <input name="embeddingTargetId" id="embeddingTargetId" required>
+  </div>
+  <div>
+    <label>
+      <input type="checkbox" name="useMapSrs">
+      Use the map SRS instead of WGS84
```

Comment:
I think we should describe this option consistently throughout the sandbox.

Reply:
What is the non map srs option, if set to false? Here it is WGS84

Reply:
It's _always_ WGS84, that's why I'm confusing that it's mentioned here explicitly. Imo mentioning the default SRS in the api docs is enough.

Reply:
okay. thanks for putting that clear. I will change it